src/test/java/teammates/test/cases/logic/FeedbackResponseCommentsLogicTest.java

```diff
@@ -85,19 +85,22 @@ public void testCreateFeedbackResponseComment() throws Exception {
         verifyPresentInDatastore(frComment);
 
         ______TS("typical successful case: frComment already exists");
+
+        FeedbackResponseCommentAttributes actualComment =
+                frcLogic.getFeedbackResponseComment(
+                        frComment.feedbackResponseId, frComment.commentGiver, frComment.createdAt);
+
+        frComment.commentText = "New Text";
         frcLogic.createFeedbackResponseComment(frComment);
-        List<FeedbackResponseCommentAttributes> actualFrComments =
-                frcLogic.getFeedbackResponseCommentForSession(frComment.courseId, frComment.feedbackSessionName);
 
-        FeedbackResponseCommentAttributes actualFrComment = null;
-        for (FeedbackResponseCommentAttributes comment : actualFrComments) {
-            if (comment.commentText.equals(frComment.commentText)) {
-                actualFrComment = comment;
-                break;
-            }
-        }
+        // check that it uses existing ID from database
```

Comment:
okay will update the comment 👍

Reply:
BTW, will the test fail if you remove the .setId() in `createFeedbackResponseComment` (i.e. make sure `frComment` has empty id before the test).

Reply:
Yup it will fail: the `.getId()` method will return `null` and an exception will be thrown. 

This is the current behaviour of the test even before the PR. The `deleteFeedbackResponseCommentById` method in the cleanup portion relies on the ID being set.